velox/functions/prestosql/aggregates/SumAggregate.h

```diff
@@ -196,6 +242,12 @@ bool registerSumAggregate(const std::string& name) {
           .intermediateType("double")
           .argumentType("double")
           .build(),
+      exec::AggregateFunctionSignatureBuilder()
+          .argumentType("DECIMAL(a_precision, a_scale)")
+          .intermediateType("DECIMAL(38, r_scale)")
```

Comment:
nit: shall we define 38 as static constexpr? Thanks!

Reply:
I feel this has better readability.